app/src/main/java/com/example/sharewhatyoucanproject/donor/DonorDrawerActivity.kt

```diff
@@ -1,11 +1,104 @@
 package com.example.sharewhatyoucanproject
 
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Intent
+import android.os.Build
 import android.os.Bundle
+import android.view.Menu
+import android.view.View
+import android.widget.TextView
 import androidx.appcompat.app.AppCompatActivity
+import androidx.drawerlayout.widget.DrawerLayout
+import androidx.navigation.findNavController
+import androidx.navigation.ui.AppBarConfiguration
+import androidx.navigation.ui.navigateUp
+import androidx.navigation.ui.setupActionBarWithNavController
+import androidx.navigation.ui.setupWithNavController
+import com.example.sharewhatyoucanproject.databinding.ActivityDonordrawerBinding
+import com.example.sharewhatyoucanproject.donor.PostsActivity
+import com.google.android.material.navigation.NavigationView
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.messaging.FirebaseMessaging
 
 class DonorDrawerActivity : AppCompatActivity() {
+    private lateinit var appBarConfiguration: AppBarConfiguration
+    private lateinit var binding: ActivityDonordrawerBinding
+    lateinit var circularProgressIndicator: com.google.android.material.progressindicator.CircularProgressIndicator
```

Comment:
shouldn't need the fully qualified import here

Reply:
done